examples/libadwaita/leaflet_sidebar.rs

```diff
@@ -58,6 +58,11 @@ impl SimpleComponent for App {
                     }
                 },
 
+                #[name = "separator"]
+                gtk::Separator {
+                    set_orientation: gtk::Orientation::Vertical,
+                },
```

Comment:
I think this would be a cleaner solution:

```suggestion
                append = &gtk::Separator {
                    set_orientation: gtk::Orientation::Vertical,
                } -> {
                    set_navigatable: false,
                },
```

Reply:
Thanks!

This is indeed better, I wasn't aware of this syntax of the view macro.

Reply:
Yeah, it's rather late in the book, no worries. It's definitely worth to check out the macro reference in the book or the macro-reference example to get an overview over what's available.

Anyway, thanks for the PR!